.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: googleapis/release-please-action@v4
+      - uses: googleapis/release-please-action@v3
```

Comment:
Just curious why we need to downgrade? Are there some bugs or it doesn't work well with our setup?

Reply:
Yeah, unfortunately. There are two issues I'm aware of.

- https://github.com/googleapis/release-please-action/issues/904
- https://github.com/googleapis/release-please-action/issues/912

We've implemented the workaround for the second issue but the first is more difficult to get around. Our release PR will always fail the DCO check. See [here](https://github.com/open-feature/python-sdk/pull/431).

Reply:
Gotcha, saw that happening on the past. Good to know what caused it, thanks!